contracts/mocks/SafeERC20Helper.sol

```diff
@@ -109,4 +109,4 @@ contract SafeERC20Helper {
     function allowance() public view returns (uint256) {
         return _succeeding.allowance(address(0), address(0));
     }
-}
+}
```

Comment:
Please add back the end-of-file newlines that were removed in SafeERC20Helper.sol, IERC20.sol, SafeERC20.sol, and package-lock.json.